source/includes/quickstart-settlement.md.erb

```diff
@@ -229,8 +229,8 @@ This applies if you are using our flexible payments (wallets) functionality.
 
 [POST https://api-sandbox.goji.investments/platformApi/settlement/investor-repayment](/#settlement-debt-post-platformapi-settlement-investor-repayment)
 
- - The system will move money from the wallet specified referenced by "sourceWalletId" to all the investors specified in the investorRepayments array.
- - It is not guaranteed that there will be as many money movements as there are investors in the investorRepayments array so your system should not rely on it.
+ - The system will move money from the wallet specified referenced by `sourceWalletId` to all the investors specified in the `investorRepayments` array.
+ - It is not guaranteed that there will be exactly as many money movements as there are investors in the `investorRepayments` array so your system should not rely on it. Reasons for this could include repayments larger than the underlying payment scheme limit or a bulk movement from the wallet to the client money account.
```

Comment:
How about something like this?

 - For any successful batch, it is guaranteed that there will be _at least one_ payment into an investor account for each entry in the `investorRepayments` array (there may be more than one payment per entry if payments are split for any reason). However, there will be exactly one INVESTMENT_PAYMENT_RECEIVED webhook per entry in the array, when the final payment arrives for that investor.
 - No guarantees of any kind are made regarding the number of _outgoing_ payments from the source wallet. Goji's systems will move the money via pass-through accounts in the most efficient way.

Reply:
That's far more eloquent than what I put so I'm changing this now 👍